doc/book/usage.md

```diff
@@ -87,6 +87,13 @@ $request = Zend\Diactoros\ServerRequestFactory::fromGlobals(
 );
 ```
 
+When no cookie array is supplied, `fromGlobals` will first try to parse the supplied `cookie` header
+before falling back to the `$_COOKIE` superglobal. This is done because PHP has some legacy handling
+for request parameters which were then registered as global variables. Due to this, cookies with a period
+in the name were renamed with underlines. By getting the cookies directly from the cookie header, you have
+access to the original cookies in the way you set them in your application and they are send by the user
+agent.
```

Comment:
If `$_COOKIE` is passed there, you get the old behaviour. Not sure if we need to discourage it though. This is a rare case which this PR fixes, and usually people will call `fromGlobals()` without arguments I suppose?